packages/web/src/apollo/suspense.tsx

```diff
@@ -145,15 +148,24 @@ const ApolloProviderWithFetchConfig: React.FunctionComponent<{
 
   // We use this object, because that's the shape of what we pass to the config.link factory
   const redwoodApolloLinks: RedwoodApolloLinks = [
+    // @MARK REMOVE: We will not need these for cookie based auth ~~~~
     { name: 'withToken', link: createTokenLink(getToken) },
     {
       name: 'authMiddleware',
       link: createAuthApolloLink(authProviderType, headers),
     },
-    // @REVIEW: Should we take this out for prod?
-    { name: 'enhanceErrorLink', link: createUpdateDataLink() },
-    { name: 'httpLink', link: createHttpLink(getGraphqlUrl(), httpLinkConfig) },
-  ]
+    // ~~~~ @END REMOVE ~~~~
+    isDev && { name: 'enhanceErrorLink', link: createUpdateDataLink() },
+    {
+      name: 'httpLink',
+      link: createHttpLink(
+        getGraphqlUrl(),
+        httpLinkConfig,
+        serverAuthState?.cookieHeader
+      ),
+    },
+    // filter boolean doesn't play nice with TS
+  ].filter((link): link is RedwoodApolloLink<any> => !!link)
```

Comment:
```suggestion
  ].filter((link): link is RedwoodApolloLinks[number] => !!link)
```

The above code works and is simple enough in this case, so go with that 🙂 

For a more generic solution I did this in `setupHelpers.ts`
```ts
// from lodash
type Truthy<T> = T extends false | '' | 0 | null | undefined ? never : T

function truthy<T>(value: T): value is Truthy<T> {
  return !!value
}

// ...

const filtered = [
  // ...
].filter(truthy)
```

Reply:
Yeah I didn't want a useless helper function haha